better-font-awesome.php

```diff
@@ -321,14 +325,20 @@ public function create_admin_page() {
         <div class="wrap">
             <?php screen_icon(); ?>
             <h2><?php echo $this->plugin_display_name; ?></h2>
-            <form method="post" action="options.php">
+            <form method="post" action="options.php" id="bfa-settings-form">
             <?php
                 // This prints out all hidden setting fields
                 settings_fields( 'option_group' );
                 do_settings_sections( self::SLUG );
-                submit_button();
-                echo $this->get_usage_text();
             ?>
+                <p>
+                    <span id="bfa-save-settings-button" class="button-primary"><?php _e( 'Save Settings', 'better-font-awesome' ); ?></span>
```

Comment:
@BraadMartin and @dashaluna - hmmmm, I'm not so sure this is the intended use. If I'm not mistaken, the `esc_html` family of functions are meant for escaping one of two things:

1. User-defined input that might contain dangerous characters/content
2. String that contain HTML

In the case you're suggesting though, and in several examples in #5 you submitted, it's not necessary to use this escaping function, since these are strings that:

1. Are hard-coded in the plugin, and thus won't contain user-defined (potentially dangerous) content (sure, the translator could throw in some malicious code, but that is something I would review and don't feel inclined to safeguard against so much)
2. Contain pure strings, without any HTML

Can you please clarify if I'm missing something?

I can see how it might be good to incorporate this for strings that *do* contain HTML, like the following:

`esc_html__( '<b>Something</b>', 'text-domain' );`

. . . but even then, I'd rather just pull out the `</b>` tags or whatever other markup is in there, so it's not editable. If the markup is error-free (which it should be, right?) and the context of the output is known (e.g. a plugin options page), then is there any reason why I would need to escape things?

I do notice that this is in place in various parts of core, but missing from others - would love to hear if you know something I don't, including links to any reading you think pertains. Thanks!